pyslet/odata2/csdl.py

```diff
@@ -622,6 +622,26 @@ class SimpleType(xsi.EnumerationNoCase):
     SimpleType.PythonType[bytes] = SimpleType.Binary
 
 
+def compare_with_none(item1, item2):
+    if item1 is None and item2 is None:
+        return 0
+    if item1 is None and item2 is not None:
+        return -1
+    if item1 is not None and item2 is None:
+        return 1
+
+    return item1 < item2
```

Comment:
Will this work? (as opposed to returning 1,-1 or 0)

Reply:
Good point. Updated